gateway_swagger.yaml

```diff
@@ -140,6 +205,9 @@ components:
           $ref: "#/components/schemas/QueryStatus"
         response:
           type: "object"
+          properties:
+            credibility:
+              $ref: "#/components/schemas/CredibilityLabel"
```

Comment:
I think this will be a part of what the policy engine gives in its response. Therefore I think we should wait on doing expansions here until the policy engine is better defined

Reply:
This was intended so we can move forward with the Browser Plugin and start using labels for each tweet. But yes, maybe it's too rushed.

While the policy engine output is being defined we can mock those labels at a Browser Plugin level and kind of ignore the API Gateway response. When a stable definition is reached then we go back to parsing the API Gateway response.